api/defaults/defaults.go

```diff
@@ -144,6 +144,12 @@ const (
 	DefaultChunkSize = 1000
 )
 
+const (
+	// DefaultMaxErrorMessageSize is the default maximum size of an error message.
+	// This can be used to truncate large error messages, which might cause gRPC messages to exceed the maximum allowed size.
+	DefaultMaxErrorMessageSize = 1024 * 100 // 100KB
```

Comment:
I know this already merged, by I wouldn't have put this in `api/defaults` - it's only ever used in `lib/srv/discovery`. Putting it in the `defaults` package makes it seem like some sort of application-wide default, but that's not the case.